pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -257,6 +308,12 @@ servingInfo:
   keyFile: /var/serving-cert/tls.key
 clusterInfo:
   masterPublicURL: https://foobar.com/api
+  monitoringURLs:
```

Comment:
I may be inclined to do:

```yaml
  monitoring:
    alertmanagerURL: https://alertmanager.url.com
    grafanaURL: https://grafana.url.com
    prometheusURL: https://prometheus.url.com
  logging:
    kibanaAppURL: https://kibanaApp.url.com
```
Just eliminates `URLs` from the wrapper object, since everything underneath has `URL` as a suffix, anyway.  I don't expect we will ever have other config, but if we had `non-url` config, we can still stick it in the same spot then.